README.md

```diff
@@ -1,32 +1,87 @@
 SJTUG mirror docker, based on tunasync/jekyll
-#Usage
+
+# Preparations
+
+Follow these steps before building the images.
+
+### Prepare tunasync tar ball
+
+Prepare a gzipped tunasync tar ball without version directory (as if generated
+by `git archive` without `--prefix`), rename it to `tunasync.tar.gz`, and
+place it into `tunasync/`.
+
+You may run
+```sh
+$ cd tunasync
+$ ../utils/genpack.sh tunasync
+```
+to generate it automatically.
+
+### Generate static mirror-web pages
+
+Suppose the static web contents are supposed to store in `/home/mirror-web`.
+
+Fetch the latest mirror-web and build the pages. You should have working
+`jekyll` installed.
+
 ```sh
-docker build -t mirror .
-docker run -d -v /your/physical-storage-for-mirror:/mnt -p 80:80 mirror bash /home/manage.sh
-# -d: daemon
-# -v: mounted volume
-# -p: mapped port
+$ utils/genpack.sh mirror-web # may generate an extra .tar.gz, just ignore it
+$ cd mirror-web
+$ jekyll build -d /home/mirror-web/_site
```

Comment:
install `jekyll` on your computer. Build the static pages, pack them and send them to the server ;-)